pkgs/applications/radio/rtl-wmbus/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  version = "unstable-2022-05-07";
+  pname = "rtl-wmbus";
+
+  src = fetchFromGitHub {
+    owner = "xaelsouth";
+    repo = pname;
+    rev = "388e7adb8e87caba2fcd50de979a6473e66d475b";
+    sha256 = "sha256-PuKoF57wUKJDZu9EwpdRP34+MBPnhEp4VKOfEE/Kog0";
+  };
+
+  preBuild = ''
+    sed -e s@/usr@@ -i Makefile
+
+    sed -e "s@^COMMIT_HASH?=.*\$@COMMIT_HASH=@" -i Makefile
+    sed -e "s@^TAG?=.*\$@TAG=$version@" -i Makefile
+    sed -e "s@^BRANCH?=.*\$@BRANCH=master@" -i Makefile
+    sed -e "s@^CHANGES?=.*\$@CHANGES=@" -i Makefile
+  '';
```

```suggestion
  makeFlags = [
    "COMMIT_HASH=${src.rev}"
    "TAG=${version}"
    "BRANCH=unknown"
    "CHANGES=unknown"
  ];

  preBuild = ''
    substituteInPlace Makefile \
      --replace "CC=gcc" "CC?=gcc" \
      --replace "/usr" ""
  '';
```

Since some replaced variables default to their environment value, they can be passed through `makeFlags` and the more common `substituteInPlace` helper can be used instead of sed.